client/src/pages/Projects.js

```diff
@@ -0,0 +1,47 @@
+import { Template } from "../components/Template";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+export const Projects = () => {
+	return (
+		<Template>
+			<Container>
+				<Row className="square rounded border border-3">
+					<Col className="col-md-6 col-sm-6 text-center py-4">
```

Comment:
Looks good, I would also test for responsiveness, you can maybe check the last few minutes of the 15 minute video I attached